plugin/src/test/groovy/org/gradle/testretry/testframework/SpockViaJUnitVintageFuncTest.groovy

```diff
@@ -31,10 +31,8 @@ class SpockViaJUnitVintageFuncTest extends SpockBaseJunit5FuncTest {
     protected String buildConfiguration() {
         return """
             dependencies {
-                implementation "org.codehaus.groovy:groovy-all:2.5.8", {
-                    exclude group: "org.junit.jupiter"
-                }
-                testImplementation "org.spockframework:spock-core:1.3-groovy-2.5"
+                implementation "org.codehaus.groovy:groovy-all:2.5.8"
+                implementation "org.spockframework:spock-core:1.3-groovy-2.5"
```

Comment:
❌ Why the change from `testImplementation` to `implementation`. That's not what users would do and hence not what we want to test here.

Reply:
Reverted